src/api.ts

```diff
@@ -543,3 +554,24 @@ function tickFeedbackCounter() {
       .info();
   }
 }
+
+/**
+* Updates the global Discord counter.
+* After each `DISCORD_COUNTER_PROMPT_AFTER` mirrord runs, displays a message asking the user to join the discord.
+*/
+function tickDiscordCounter() {
+  const previousRuns = parseInt(globalContext.globalState.get(DISCORD_COUNTER) ?? '0');
+  const currentRuns = previousRuns + 1;
+
+  globalContext.globalState.update(DISCORD_COUNTER, currentRuns);
+
+  if ((currentRuns % DISCORD_COUNTER_PROMPT_AFTER) === 0) {
+    new NotificationBuilder()
+      .withMessage(`Need any help with mirrord? Come chat with our team on Discord!`)
+      .withGenericAction("Join us!", async () => {
+        vscode.commands.executeCommand(MirrordStatus.joinDiscordCommandId);
+      })
+      .withDisableAction('promptReview')
```

Comment:
This is the same identifier as in `tickFeedbackCounter`, it should be unique. Otherwise disabling one of the prompts will disable another.
I should've provided a doc to this, sorry :sweat_smile:

Reply:
ahh I missed one! no worries